### PR TITLE
[Feature] Merged Resiliance and Reliability Tables to one Dashboard

### PR DESCRIPTION
**Pull request by Vanessa-Kris**
- Merged reliability and resilience dashboards
- Edited ReadMe
- Created 404 page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
+Welcome to SMSWithoutBorders Dashboard, a powerful tool designed to measure the reliability of gateway clients ans available gateway servers.
 
-## Installation
+## Getting Started
 
-`yarn install`
+To get started with the SMSWithoutBorders Dashboard, follow these simple steps:
 
-`yarn start`
+1. **Create .env File**: Create a `.env` file in the root directory of the project. This file will contain environment variables necessary for the application to function correctly.
```
Comment (PromiseFru) [inline README.md:9]: Add the required environment variables needed for the dashboard to work.

File: src/Components/DateSearch.js [hunk 2]
```diff
@@ -2,23 +2,29 @@ import React, { useState, useEffect } from "react";
 import TextField from "@mui/material/TextField";
 import Stack from "@mui/material/Stack";
 import Autocomplete from "@mui/material/Autocomplete";
-import { fetchData } from "../Utils/FetchData";
+// import { fetchData } from "../Utils/FetchData";
```
Comment (PromiseFru) [inline src/Components/DateSearch.js:5]: Remove unnecessary comments.

File: src/Components/DateSearch.js [hunk 3]
```diff
@@ -2,23 +2,29 @@ import React, { useState, useEffect } from "react";
...
-			.catch((error) => {
-				console.error("Error fetching data:", error);
-			});
-	}, []);
+			} catch (error) {
+				console.error("Error fetching dates:", error);
+			}
+		};
+
+		fetchDates(); // Fetch dates on component mount
```
Comment (PromiseFru) [inline src/Components/DateSearch.js:26]: Remove unnecessary comments.

File: src/Pages/Dashboard.js
Comment (PromiseFru) [inline src/Pages/Dashboard.js:1]: Remove unused files.

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Thank you @Vanessa-Kris! Please see my comments,

**Comment by PromiseFru** [comment]
@Vanessa-Kris, please update the PR title to reflect the change being introduced. For instance: [Feature] Add table filter functionality.

**Comment by Vanessa-Kris** [comment]
@PromiseFru , thank you for the review.  I've made the requested changes.

File: README.md [hunk 4]
```diff
@@ -1,19 +1,202 @@
...
-`yarn install`
+To get started with the SMSWithoutBorders Dashboard, follow these simple steps:
 
-`yarn start`
+1. **Create .env File**: Create a `.env` file in the root directory of the project. Add variables:
 
-## Tools
+```json
+REACT_APP_RELIABILITY_URL=https://REACT_APP_RELIABILITY_URL
+REACT_APP_RESILIENCE_URL=https://REACT_APP_RESILIENCE_URL
```
Comment (PromiseFru) [inline README.md:13]:
```suggestion
REACT_APP_RELIABILITY_URL=# reliability data source (replace with actual URL)
REACT_APP_RESILIENCE_URL=# resilience data source (replace with actual URL)

> [!NOTE]
> Please ensure to replace the placeholders with real URLs pointing to your reliability and resilience data sources.
```

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Thank you, @Vanessa-Kris! Just a little more cleanup! 😉

**Review by PromiseFru — APPROVED** [review]

**Pull request merged**